.github/workflows/arizona-bootstrap-release.yml

```diff
@@ -0,0 +1,25 @@
+name: Create PR for new Arizona Bootstrap Release
+on:
+  repository_dispatch:
+    types: az_bootstrap_release
+
+jobs:
+  release:
+    name: Create PR
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        # with:
+        #   ssh-key: ${{ secrets.SELF_DEPLOY_KEY }}
+
+      - name: Create new branch and PR
+        run: |
+          git checkout -b 'az-bootstrap-${{ github.event.client_payload.version }}'
+          jq '.require."az-digital/arizona-bootstrap" |= "${{ github.event.client_payload.version }}"' composer.json > composer.json.new
+          mv composer.json.new composer.json
+          git add composer.json
```

Comment:
We'll also need to update the `AZ_BOOTSTRAP_STABLE_VERSION` constant in the [themes/custom/az_barrio/includes/common.inc](https://github.com/az-digital/az_quickstart/blob/910edb8d374008c1f76e3450a598ded5e773e549/themes/custom/az_barrio/includes/common.inc#L10) file.